src/routes/LandingPage/LandingPage.tsx

```diff
@@ -0,0 +1,92 @@
+import { components } from '@/api/schema';
+import { MainLayout, Title } from '@/components';
+import { useLoginLink } from '@/hooks/useLoginLink';
+import { Button } from '@jod/design-system';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { MdArrowForward } from 'react-icons/md';
+import { NavLink, useLocation, useNavigate, useRouteLoaderData } from 'react-router';
+
+const LandingPage = () => {
+  const { i18n } = useTranslation();
+  const { t } = useTranslation();
```

Comment:
You should use the hook only once `const { t, i18n } = useTranslation()`. Since you only need the `language` from `i18n` you can destructure it:
`const { t, i18n: { language } } = useTranslation()`, then you can just use `language` instead of `i18n.language`

Reply:
Oops 🫠 Fixed!